lib/resources/example_json/r4_examples_health_cards.rb

```diff
@@ -43,7 +49,7 @@ module Resources
             },
             {
               'name': 'hostedResource',
-              'valueUri': 'https://fhir-myrecord.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Patient/12742542'
+              'valueUri': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Patient/12742542'
```

Comment:
More curious-  Why’d we switch to open?

Reply:
Oops, my bad. Forgot to rename those. Updated here https://github.com/cerner/fhir.cerner.com/pull/687/commits/9925f7f4c1dad4ed8efbea76cfeaef8042104a80